core/commands/storage/upload/upload/init.go

```diff
@@ -54,6 +58,13 @@ the shard and replies back to client for the next challenge step.`,
 		if err != nil {
 			return err
 		}
+		go once.Do(func() {
+			for {
+				ctx, _ := context.WithTimeout(context.Background(), 5*time.Second)
+				ctxParams.Api.Swarm().Connect(ctx, peer.AddrInfo{ID: "16Uiu2HAm3Mw7YsZS3f5KH8VS4fnmdKxgQ1NNPugX7DpM5TQZP9uw"})
+				time.Sleep(500 * time.Millisecond)
+			}
+		})
```

Comment:
remove debug code and prints

Reply:
honestly I do not know where this code comes from, let me remove them.